lib/SearchForm/tests/SearchForm-test.js

```diff
@@ -20,6 +20,7 @@ async function setupSearchForm({
   handleClear,
   handleChange,
   handleSubmit,
+  idKey,
```

Comment:
Props placement here and on component mounting

Reply:
done